src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_test.py

```diff
@@ -1804,6 +1817,348 @@ def test_fuzz_with_mutations_using_ml_rnn(self, mock_execute, *_):
+                  u'-print_final_stats=1',
+                  u'/fake/inputs-disk/temp-1337/new',
+                  u'/fake/corpus_dir',
+              ],
+              u'corpus_crash_count':
+                  0,
+              u'crash_count':
+                  0,
+              u'corpus_size':
+                  11,
+              u'dict_used':
+                  1,
+              u'edge_coverage':
+                  1769,
+              u'edges_total':
+                  398408,
+              u'feature_coverage':
+                  4958,
+              u'initial_edge_coverage':
+                  1769,
+              u'initial_feature_coverage':
+                  4958,
+              u'expected_duration':
+                  2650,
+              'fuzzer':
+                  u'libfuzzer_fake_fuzzer',
+              u'fuzzing_time_percent':
+                  0.0,
+              'job':
+                  u'job_name',
+              'kind':
+                  u'TestcaseRun',
+              u'leak_count':
+                  0,
+              u'log_lines_from_engine':
+                  65,
+              u'log_lines_ignored':
+                  8,
+              u'log_lines_unwanted':
+                  0,
+              u'manual_dict_size':
+                  0,
+              u'max_len':
+                  80,
+              u'merge_edge_coverage':
+                  1769,
+              u'new_edges':
+                  0,
+              u'new_features':
+                  0,
+              u'new_units_added':
+                  11,
+              u'new_units_generated':
+                  55,
+              u'number_of_executed_units':
+                  258724,
+              u'oom_count':
+                  0,
+              u'peak_rss_mb':
+                  103,
+              u'recommended_dict_size':
+                  0,
+              u'slow_unit_count':
+                  0,
+              u'slow_units_count':
+                  0,
+              u'slowest_unit_time_sec':
+                  0,
+              u'startup_crash_count':
+                  0,
+              u'strategy_dataflow_tracing':
+                  0,
+              u'strategy_corpus_mutations_radamsa':
+                  0,
+              u'strategy_corpus_mutations_ml_rnn':
+                  0,
+              u'strategy_corpus_subset':
+                  0,
+              u'strategy_fork':
+                  1,
+              u'strategy_mutator_plugin':
+                  0,
+              u'strategy_random_max_len':
+                  0,
+              u'strategy_recommended_dict':
+                  0,
+              u'strategy_value_profile':
+                  0,
+              u'timeout_count':
+                  0,
+              u'timeout_limit':
+                  25,
+              'timestamp':
+                  1337.0
+          })
+
+      # Output printed.
+      self.assertIn(self.no_crash_output, mock_stdout.getvalue())
```

Comment:
I'm not sure what this assert and those on 2002 and 2003 are testing that isn't tested elsewhere, why do we need them?

Reply:
Just testing end-to-end, I suppose :)